general_schema.json

```diff
@@ -128,6 +128,10 @@
           "items": {
             "type": "string",
             "description": "List of synonyms"
+          },
+          "author_annotation_fields": {
```

Comment:
seems nesting is wrong, this emergent is inside `synonyms`.

Also, Should this be an array of objects? Such as:
`
"author_annotation_fields": {
            "type": "array",
            "description": "A set of author defined key value pairs annotating the cell set. The names and aims of these fields MUST not clash with annotationThis schema accepts author defined fields.",
            "items": {
                  "type": "object",
                  "description": "Author defined key value pairs annotating the cell set. The names and aims of these fields MUST not clash with annotationThis schema accepts author defined fields."
             }
          }
`